.ci/matrix-runtime-javas.yml

```diff
@@ -6,11 +6,8 @@
 # or 'openjdk' followed by the major release number.
 
 ES_RUNTIME_JAVA:
-  - java8
-  - java8fips
```

Comment:
Shouldn't we have an alternative before removing this?
There's #37250 so this will not fall trough the cracks, but in the interim we will have no fips testing,
and fips might have to play catch-up once we get the tests back. 
I think that's something we should generally try to avoid in CI.

Reply:
I discussed this with @jkakavas and he will be working on getting a FIPS-compliant JDK 11 ready, but agreed that can happen independently. He will focus on that in the upcoming week.